proposals/csharp-7.0/tuples.md

```diff
@@ -399,39 +451,67 @@ static class Extensions
 }
 ```
 
-Overload resolution for `Deconstruct` methods considers only the arity of the `Deconstruct` method. If multiple `Deconstruct` methods of the same arity are accessible, the expression is ambiguous and a binding time error occurs.
+**ISSUE:** I think we need to say more about this code (which is not an example, but is intended to show/say just what is going on/needed).
+
+Overload resolution for `Deconstruct` methods considers only the arity of the `Deconstruct` method. If multiple `Deconstruct` methods of the same arity are accessible, the expression is ambiguous and a binding-time error shall occur.
```

Comment:
Regarding the "arity" definition. This paragraph implies that names aren't part of "arity".

Reply:
OK.